content/client-lib-development-guide/features.textile

```diff
@@ -2141,6 +2145,10 @@ class VCDiffDecoder
 class DeltaExtras
   from: String // the id of the message the delta was generated from
   format: String //the delta format. Only vcdiff is supported as at API version 1.2
+
+class InteractionExtras
```

Comment:
We should also update `TM2i` with something similar to the sentence about `DeltaExtras`